app/views/password_resets/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% provide :title, t(".reset_passwd") %>
+<h1><%= t ".reset_passwd" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: password_reset_path(params[:id]) do |f| %>
+      <%= render "shared/error_messages" %>
+      <%= hidden_field_tag :email, @user.email %>
+      <%= f.label t ".password" %>
+      <%= f.password_field :password, class: "form-control" %>
+      <%= f.label t ".confirm" %>
```

Comment:
Theo em thì:
Chẳng hạn như trong file này của em thì nếu để `:password_confirmation` sẽ sinh ra:
`<input id="user_password_confirmation" ... class="..."
<label for="user_password_confirmation" ...`
còn nếu không có thì sẽ sinh ra:
`<input id="user_password_confirmation" ... class="..."
<label for="user_password_Confirmation" ...` 
(`Confirmation` vì rails không thấy có :password_confirmation nên sẽ dùng cái label i18n)

Và mấu chốt là ở attribute `id` cho `input` và `for` cho `label`, 2 cái này phải giống nhau thì mới sử dụng được effect đúng ý mình ạ